classes/03class/exercises/c03-docker01/alanlima/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:latest
```

Comment:
Great docker file,
Nice use of the Environment variables.

A quick note, it's sometimes not wise to use the latest version as it can contain some bugs when it's updated by the provider.
Best practice is to check the tag via dockerhub and specify the version!

Reply:
Thanks for the heads up @starkmatt 
I will update the Dockerfile with the current alpine version.